### PR TITLE
GUI-1680 Correct error by improving validation

### PR DESCRIPTION
**Pull request by coyote240**
https://eucalyptus.atlassian.net/browse/GUI-1680

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1159/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1163/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1167/
Test PASSed.

File: eucaconsole/templates/managecredentials.pt [hunk 1]
```diff
@@ -46,39 +46,78 @@
...
+                                <div class="small-12 columns field">
+                                    <div class="controls-wrapper">
+                                        <label class="left">${changepassword_form['new_password'].label.text}&nbsp;<span class="req">*</span></label>
+                                    </div>
+                                    <div password-strength="" class="password-strength" ng-class="strengthLabel()">
+                                        ${structure:changepassword_form['new_password'](autocomplete="off", ngModel="new_password", strength="", required="", pattern="^.{6,}$", noMatch="current_password")}
+                                        <span class="strength"></span>
+                                    </div>
+                                    <div class="client-validation">
+                                        <span class="error bad-pattern">Password must be a minimum of six characters long.</span>
```
Comment (kamalgill) [inline eucaconsole/templates/managecredentials.pt]: @coyote240 : We should add an i18n:translate attribute here

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1178/
Test PASSed.

**Pull request merged**